plugins/woocommerce-blocks/assets/js/blocks/checkout/test/block.js

```diff
@@ -164,4 +163,26 @@ describe( 'Testing cart', () => {
 
 		expect( fetchMock ).toHaveBeenCalledTimes( 1 );
 	} );
+
+	it( 'Ensures checkbox labels have unique IDs', async () => {
+		allSettings.checkoutAllowsGuest = true;
+		allSettings.checkoutAllowsSignup = true;
+		dispatch( CHECKOUT_STORE_KEY ).__internalSetCustomerId( 0 );
```

Comment:
You must reset those values to their original values at the end of the your test to not pollute other tests. Each test should leave the global state like it was found.

Reply:
Also reset `allSettings.checkoutAllowsGuest , allSettings.checkoutAllowsSignup` to their default values.